packages/web-client/src/webdav/listFiles.ts

```diff
@@ -1,12 +1,33 @@
+import { buildResource } from 'files/src/helpers/resources'
+import { DavProperties, DavProperty } from 'web-pkg/src/constants'
 import { OwnCloudSdk } from 'web-pkg/src/services'
 import { isPublicSpaceResource, Resource, SpaceResource } from '../helpers'
 
-export const ListFilesFactory = (sdk: OwnCloudSdk) => {
+export const GetListFilesFactory = (sdk: OwnCloudSdk) => {
```

Comment:
Oh, doh - I tried to stick to the naming scheme of GetCapabilitiesFactory, but the Get did not refer to Factory there hehe. Let's revert this naming change tomorrow...